packages/edit-site/src/components/navigation-sidebar/index.js

```diff
@@ -23,8 +23,12 @@ export default function NavigationSidebar() {
 	return (
 		<>
 			<NavigationToggle isOpen={ isNavigationOpen } />
-			<NavigationPanel isOpen={ isNavigationOpen } />
-			<NavigationPanelPreviewSlot />
+			{ isNavigationOpen && (
```

Comment:
Do we know if this affects performance much? As an example, I think that the block inserter area has poor performance partly because it re-mounts every time you open it, so all the expensive calculations happen all over again.

But as a short-term measure this is probably fine!

Reply:
Im sure it does a little.. not too concerned though as Im really just trying to consistently reproduce and pinpoint these bugs that are horribly inconsistent and play RNG troll games with us.

Reply:
In the spirit of keeping performance, I updated this so that the component stays mounted after the first time it is triggered.